scripts/bash/run_enceladus.sh

```diff
@@ -170,6 +174,11 @@ validate "$DATASET_NAME" "--dataset-name"
 validate "$DATASET_VERSION" "--dataset-version"
 validate "$REPORT_DATE" "--report-date"
 
+if [[ "$DEPLOY_MODE" != "client" ]] && [[ "$DEPLOY_MODE" != "cluster" ]]; then
```

Comment:
Is this meant to be checked against master instead?

Reply:
Indeed

Reply:
Thanks a lot for spotting this, btw!